tests/testsuite/test_predicates.py

```diff
@@ -251,39 +267,42 @@ def test_one_arg(self):
         def p(a=None, *args, **kwargs):
             assert len(args) == 0
             assert len(kwargs) == 0
-            assert a == 'a'
+            assert a == "a"
+
         assert p.num_args == 1
-        p.test('a')
+        p.test("a")
 
     def test_two_args(self):
         @predicate
         def p(a=None, b=None, *args, **kwargs):
             assert len(args) == 0
             assert len(kwargs) == 0
-            assert a == 'a'
-            assert b == 'b'
+            assert a == "a"
+            assert b == "b"
+
         assert p.num_args == 2
-        p.test('a', 'b')
+        p.test("a", "b")
 
     def test_no_mask(self):
         @predicate
         def p(a=None, b=None, *args, **kwargs):
             assert len(args) == 0
             assert len(kwargs) == 1
-            'c' in kwargs
```

Comment:
What was I thinking here? :)

Reply:
would not have spotted it for a million years. flake8 FTW :)